tool-populate/index.js

```diff
@@ -104,24 +104,44 @@ function destroyObjects(objects) {
   }
 }
 
+function setRoleACLs() {
```

Comment:
I've updated, removing these two lines:
```
acl.setPublicReadAccess(false)
acl.setPublicWriteAccess(false)
```
You are right about the redundancy of setting the publicWrite. The public read rights are set in the `createRole` function [here](https://github.com/ecohealthalliance/GoodQuestion/blob/21ea47337ddf2453a05da1708e37d6c535429050/tool-populate/api/Roles.js#L28)